### DISCUSSION TITLE
New logo for UAD-ng

### DISCUSSION THREAD

**Discussion by AnonymousWP** [Universal-Debloater-Alliance/universal-android-debloater-next-generation#164] [Polls]
Because we still don't have a logo yet, here are some logos. It'll be appearing on the repo but will also be part of the software as an icon.

Logo 1:

![Senza_titolo_189_20240107193532](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/50231698/4df5fff4-1d2d-404c-b1c1-a42282ebc2a2)

Logo 2:

![20240107_105904_0000](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/50231698/e3d428a3-31f3-48d5-9a63-d19cbc8dc3fb)

Logo 3:

![20240107_110410_0000](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/50231698/9c355605-7695-4d69-be3f-16cb6318cbce)

Logo 4:

![20240107_110443_0000](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/50231698/30d96b9f-f547-47c5-b045-a7e7456f4271)

Logo 5:

![20240107_110726_0000](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/50231698/de83e2b9-67ed-406c-87b7-e61a465e9da8)

Logo 6:

![20240107_110848_0000](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/50231698/d341b1a6-dd1a-4d30-92be-be2da1b530fb)

**Comment by AnonymousWP** [comment]
Someone from our Discord server made a design:

![image](https://github.com/Universal-Debloater-Alliance/universal-android-debloater-next-generation/assets/50231698/61b4dfad-4dfc-47d1-a5c1-3b119d708adf)

**Reply by Obegg** [reply]
I like the black variant of that

**Comment by adhirajsinghchauhan** [comment]
Logo will be displayed as the taskbar icon, so it needs to be simple. If it has text, it needs to in such a way that it remains legible when scaled down to whatever the platform default size is. **So "UADng" would not be legible, while "UAD" on the first line and "ng" below it would be.**

I perceive this poll to be merely to gather feedback, ultimately we'll have to choose something after comparing how it looks like on each platform. So it may not be the most-voted one.

**Reply by adhirajsinghchauhan** [reply]
Oh and background should probably be transparent. I don't think I've ever seen any logo (in this context) with a coloured background.

**Comment by AnonymousWP** [comment]
Our logo is now set in both the repo and UAD-ng.
